sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -139,6 +139,8 @@ def __init__(
         self._last_received_event: Optional[EventData] = None
         self._receive_start_time: Optional[float] = None
 
+        super(EventHubConsumer, self).__init__()
```

Comment:
any reason this was added in?

Reply:
because I put a init in the base class to fix some typing warnings